code/modules/projectiles/guns/energy/nuclear.dm

```diff
@@ -35,6 +35,26 @@
 		else
 			user.update_inv_r_hand()
 
+/obj/item/weapon/gun/energy/gun/cyborg
+	desc = "An energy-based laser gun that draws power from the cyborg's internal energy cell directly. So this is what freedom looks like?"
+
+/obj/item/weapon/gun/energy/gun/cyborg/process()
+	return 1
+
+/obj/item/weapon/gun/energy/gun/cyborg/process_chambered()
+	if(in_chamber)
+		return 1
+	if(isrobot(src.loc))
+		var/mob/living/silicon/robot/R = src.loc
+		if(R && R.cell && R.cell.charge >= 83)
+			R.cell.use(83)
+			in_chamber = new projectile_type(src)
+			return 1
+	return 0
+
+/obj/item/weapon/gun/energy/laser/cyborg/emp_act()

```

Comment:
Do eguns have a chance of blowing up in emps? That's the only reason I can see for that code to be there. Anyways it's not much and helps with the consistency of the cyborg laser gun.